src/js/components/DropButton/DropButton.js

```diff
@@ -9,7 +9,7 @@ const defaultDropAlign = { top: 'top', left: 'left' };
 const DropButton = forwardRef(
   (
     {
-      a11yTitle = 'Open Drop',
+      a11yTitle,
```

Comment:
```suggestion
      a11yTitle = 'Open Drop',
```
Can we leave this here as a default prop? instead of adding it below.

Reply:
yes looks cleaner!